.github/workflows/build-test.yml

```diff
@@ -29,7 +29,8 @@ jobs:
         with:
           path: ${{ steps.yarn-cache-dir.outputs.YARN_CACHE_DIR }}
           key: yarn-cache-${{ runner.os }}-${{ steps.yarn-version.outputs.YARN_VERSION }}-${{ hashFiles('yarn.lock') }}
-      - run: yarn --frozen-lockfile
+      # TODO: Remove ignore-engines when eslint-plugin-jsdoc has been upgraded
```

Comment:
Hmm, it seems that we may want to bump this package first before merging this PR. The fact that we're seeing this error means we would also see this error in development too.

Reply:
Alright, but in the meantime, when I go to this branch and run `yarn install`, I see an error because of this package. It seems that not being able to install dependencies would be confusing for people who want to work on this library, no?

Reply:
Okay, cool. FWIW I think it's silly that Yarn ignores `node.engines`, but I think we've had that discussion before and I guess we can use https://github.com/devoto13/yarn-plugin-engines if we really feel like it. I'm good with your change.